dagger/dag_creator/airflow/operator_creators/dbt_creator.py

```diff
@@ -0,0 +1,55 @@
+from dagger.dag_creator.airflow.operator_creator import OperatorCreator
+from airflow.operators.bash import BashOperator
+
+
+class DbtCreator(OperatorCreator):
+    ref_name = "dbt"
+
+    def __init__(self, task, dag):
+        super().__init__(task, dag)
+
+        self._project_dir = task.project_dir
+        self._profile_dir = task.profile_dir
+        self._profile_name = task.profile_name
+        self._select = task.select
+
+    def _generate_deps_command(self):
+        command = [
+            "dbt deps",
+            f"--project-dir {self._project_dir}",
+            f"--profiles-dir {self._profile_dir}",
+        ]
+
+        return " ".join(command)
+
+    def _generate_build_command(self):
+        command = [
+            "dbt build",
+            f"--project-dir {self._project_dir}",
+            f"--profiles-dir {self._profile_dir}",
+            f"--target {self._profile_name}",
+        ]
+
+        if self._select:
+            command += [f"--select {self._select}"]
+
+        dbt_vars = ",".join(
+            [f"{param_name}:{param_value}" for param_name, param_value in self._template_parameters.items()]
```

Comment:
dbt vars are exactly built as required:
https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-on-the-command-line

We need to add quotes and curly brackets, and handle any eventual quotes that are used in the arguments params.

Reply:
Thanks, I found this doc: https://docs.getdbt.com/docs/build/project-variables#defining-variables-on-the-command-line, changed the code to generate the second version. Without quotes. If escape needed, we can add the escape to the parameter itself. But I would be surprised.